src/index.css

```
@@ -1 +1,53 @@
+@theme {
+  --color-button: #724cf9;
+  --color-button-hover: #6b4ecf;
+  --color-button-clicked: #573eb0;
+
+  --color-sad-gray: rgb(179, 179, 179);
+
+  --color-error-red: rgb(199, 33, 33);
+}
+
+@plugin "daisyui" {
+  themes: abyss, pastel;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)